modules/installation-user-infra-machines-iso.adoc

```diff
@@ -43,61 +47,47 @@ You must download images with the highest version that is less than or equal
 to the {product-title} version that you install. Use the image versions
 that match your {product-title} version if they are available.
 Only use ISO images for this procedure.
-RHCOS qcow2 images are not supported for bare metal installs.
+{op-system} qcow2 images are not supported for bare metal installs.
 ====
 +
-You must download the ISO file and the RAW disk file.
-Those file names resemble the following examples:
+ISO file names resemble the following example:
 
-** ISO: `rhcos-<version>-installer.<architecture>.iso`
-** Compressed metal RAW: `rhcos-<version>-metal.<architecture>.raw.gz`
+`rhcos-<version>-installer.<architecture>.iso`
 endif::openshift-origin[]
 ifdef::openshift-origin[]
 . Obtain the {op-system} images from the
 link:https://getfedora.org/en/coreos/download?tab=metal_virtualized&stream=stable[{op-system} Downloads] page
 endif::openshift-origin[]
 
-. Upload either the RAW {op-system} image file to your HTTP server and
-note its URL.
-+
-[IMPORTANT]
-====
-If you plan to add more compute machines to your cluster after you finish
-installation, do not delete these files.
-====
-
 . Use the ISO to start the {op-system} installation. Use one of the following
 installation options:
 ** Burn the ISO image to a disk and boot it directly.
 ** Use ISO redirection via a LOM interface.
 
-. After the instance boots, press the `TAB` or `E` key to edit the kernel command line.
-. Add the parameters to the kernel command line:
+. Boot the ISO image. You can interrupt the installation boot process to
+add kernel arguments. However, for this ISO procedure we recommend you use
```

Comment:
s/we recommend you use/it is recommended that you use/

Reply:
Done.